ubuntu-16/oneapi/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM axom/compilers:ubuntu-base
+FROM axom/compilers:gcc-8
```

Comment:
This brings up an issue that we lost information about which ubuntu this compiler tag is using.  For a follow up PR though.